packages/dashboard/src/customization/widgets/types.ts

```diff
@@ -110,6 +110,13 @@ export type LineProperties = {
   thickness?: number;
 };
 
+export type TriangleProperties = {
+  borderStyle?: 'solid' | 'dashed' | 'dotted';
```

Comment:
I wonder if we can specify something a little more generic regarding symbol stylings? Are these the same properties included on rectangle and line?

Reply:
i think the line is a little different from other symbols. for example, it doesn't have a fill, and it will not have a rotation field if it is ever added to other symbols in the future. we should keep the line separate.

but i do agree that other shapes, like the rectangle, triangle, and ellipse can have the same set of properties.

Reply:
Cool, ya I think that's a good idea!